scripts/tests/test_suites.sh

```diff
@@ -141,9 +226,9 @@ for j in "${iter_array[@]}"; do
             cat <(timeout 1 dns-sd -B _matterc._udp)
         fi
         echo "          * Pairing to device"
-        "${test_case_wrapper[@]}" out/debug/standalone/chip-tool pairing qrcode "$node_id" MT:D8XA0CQM00KA0648G00 | tee "$pairing_log_file"
+        "${test_case_wrapper[@]}" ${tool_run_prefix} out/debug/standalone/chip-tool pairing qrcode "$node_id" MT:D8XA0CQM00KA0648G00 | tee "$pairing_log_file"
```

Comment:
I'm not sure about the ordering here.  The use case for test_case_wrapper is something like `gdb`.  Do we really want `gdb sudo ...` as opposed to `sudo ip netns exec tool gdb ...`?

Reply:
I did not know what the wrapper does. It makes sense to switch order, will do so.